update: list of most problematic setup requirements in onboarding tutorials (#251)
JetBrains/kotlin-multiplatform-dev-docs: topics/compose-onboard/compose-multiplatform-create-first-app.md

Check out the article about [setting up an environment for Kotlin Multiplatform development](multiplatform-setup.md),

> if you haven’t already.

Reviewer: ```suggestion
if you haven't already.
```

Make sure that you have:

* Installed the [Kotlin Multiplatform plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform) for Android Studio.

> * Launched Xcode at least once and accepted the terms of use if you're planning to build iOS apps.

Reviewer: ```suggestion
* Launched Xcode at least once and accepted the terms of use if you plan to build iOS apps.
```

> * Ran Kdoctor to check for any issues in the setup.

Reviewer: ```suggestion
* Run Kdoctor to check for any issues in the setup.
```
